src/groundStation/groundStation.py

```diff
@@ -316,13 +312,10 @@ def receive(self):
                 print(rxData)
 
     def handlePipeMode(self):
-        if self.uTrns_enable == True:
-            if (time.time() - self.uTrns.last_tx_time) > self.uTrns.pipetimeout_s:
-                self.uTrns.enterPipeMode()
-                #may need to add delay here?
-                command, port, toSend = self.getInput(inVal= self.satellite +'.general.UHF_IS_IN_PIPE_NOTIFICATION(1)')
+        if self.uTrns is not None:
+            if self.uTrns.handlePipeMode():
```

Comment:
This and the next line are removed as per tganley/bugfix/flatsat-using-sdr. @tganley can you open a PR for that asap?

Reply:
We seem to be slipping back to the old ways with the flatsat-testing branches...

Reply:
When I figure out why this is failing tomorrow I'll merge this branch with this change anyway. These long lived branches with "little" fixes all over the place are difficult to work with